content/aips/SUPPLY-BORROW-CAPS-UPDATE-AAVE-V3-02-12-2023.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Chaos Labs Supply and Borrow Cap Updates - Aave V3 Polygon and Arbitrum - 2023.02.07
+shortDescription: Update Supply and Borrow Caps on Polygon, Arbitrum on Aave V3 
+author: Chaos Labs (@ori-chaoslabs, @yhayun)
+discussions: https://governance.aave.com/t/arc-chaos-labs-supply-and-borrow-cap-updates-aave-v3-polygon-and-arbitrum-2023-02-07/11605
+created: 2023-2-12
+---
+
+# Simple Summary
+
+A proposal to adjust five (11) total risk parameters, including Supply Cap and Borrow Cap, across seven (7) Aave V3 Polygon and Arbitrum assets.
+
+
+# Motivation
+
+The objective of this proposal is to recommend updated supply and borrow caps for V3 assets that have reached high utilization of either cap. The recommendations provided in this proposal were derived using Chaos Labs’ Updated Supply and Borrow Cap Methodology
+
+The respective governance forum discussion is linked below:
+- [Chaos Labs Supply and Borrow Cap Updates - Aave V3 Polygon and Arbitrum - 2023.02.07](https://governance.aave.com/t/arc-chaos-labs-supply-and-borrow-cap-updates-aave-v3-polygon-and-arbitrum-2023-02-07/11605)
+
+
+# Specification
+
+The following risk parameter proposal is presented below:
+
+- Arbitrum
```

Comment:
Would it be possible for you to use `markdown tables` instead of images?
Not a must, but working a lot more reliable across different uis: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#creating-a-table

Reply:
fixed